bee-common/bee-packable/tests/bounded.rs

```diff
@@ -51,9 +51,13 @@ macro_rules! impl_packable_test_for_bounded_integer {
         #[test]
         fn $packable_invalid_name() {
             let bytes = vec![0u8; core::mem::size_of::<$wrapped>()];
-            let unpacked = <$wrapper>::unpack_from_slice(&bytes);
+            let unpacked = <$wrapper>::unpack_verified(&bytes);
 
-            assert!(matches!(unpacked, Err(UnpackError::Packable($error(0)))))
+            assert!(
+                matches!(unpacked, Err(UnpackError::Packable($error(0)))),
+                "found {:?}",
```

Comment:
Was this debug or ... ?

Reply:
it was debug but maybe we should do it in every `assert!(matches!(...))`.

Reply:
Alright, I wonder to what extent it's useful if we're expecting an error

Reply:
I did it because I changed the errors and couldn't figure out the right variants. But future PR maybe.